content/posts/newsletter-016/index.md

```diff
@@ -67,6 +67,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [rib]
+
+![rib](rib.gif)
+
+Parsing a 3D model file and understand the different links between bone matrices,
```

Comment:
`understand` -> `understanding`?

Note that this change would take the line over 80 chars, which will make the linter unhappy, so you might need to redo the line breaks if you make this change - apologies for that!

Reply:
of course! thank you for the typo